PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+                               ("updateFieldGroups", []),
+                               ("deductible", deductible),
+                               ("lumpSumPayment", lumpSumPayment),
+                               ("removeIndemnityPerEvent", False),
+                               ("removeIndemnityPerItem", False)])
+
+    exposureUpdateRequest = dict([("addFieldGroups", []),
+                                  ("addPerils", []),
+                                  ("endPerils", []),
+                                  ("exposureLocator", exposureLocator),
+                                  ("fieldValues", {}),
+                                  ("removeFieldGroups", []),
+                                  ("updateFieldGroups", []),
+                                  ("updatePerils", [perilUpdateRequest])])
+
+    # exposureUpdateRequest = dict([("addFieldGroups", []),
+    #                               ("addPerils", []),
+    #                               ("endPerils", [perilLocatorToRemove]),
+    #                               ("exposureLocator", exposureLocator),
+    #                               ("fieldValues", {}),
+    #                               ("removeFieldGroups", []),
+    #                               ("updateFieldGroups", []),
+    #                               ("updatePerils", [])])
+
+
+    # exposureUpdateRequest = dict([("addFieldGroups", []),
+    #                               ("addPerils", [{"name":"bodily_injury"}]),
+    #                               ("endPerils", []),
+    #                               ("exposureLocator", exposureLocator),
+    #                               ("fieldValues", {}),
+    #                               ("removeFieldGroups", []),
+    #                               ("updateFieldGroups", []),
+    #                               ("updatePerils", [])])
+
+    policyUpdateRequest = dict([("addExposures", []),
+                                ("addFieldGroups", []),
+                                ("endExposures", []),
+                                ("fieldValues", {}),
+                                ("removeFieldGroups", []),
+                                ("updateExposures", [exposureUpdateRequest]),
+                                ("updateFieldGroups", []),
+                                ("hasPolicyUpdates", False)])
+
+    endorsementRequest = dict([("endorsementName", "generic"),
+                               #("startTimestamp", 1564642800000),
+                               ("startTimestamp", 1565852400000),
+                               ("updatePolicy", policyUpdateRequest)])
+
+    endorsementURL = "https://api.develop.socotra.com/policy/100005202/endorse"
+    r = requests.post(endorsementURL, json=endorsementRequest, headers=headers)
+    print(r.headers.items())
+    #print(r)
+
+    updatedPolicy = requests.get(policyResponseURL, headers=headers)
+    updatedData = updatedPolicy.json()
+    print(updatedData)
+
```

Comment:
you need an extra space here before the `if __name__ == '__main__':` line to appease the linter.